evgMrmApp/Db/evgMxc.db

```diff
@@ -68,7 +68,7 @@ record(longin , "$(P)Prescaler-RB") {
     field( DTYP, "Obj Prop uint32")
     field( INP , "@OBJ=$(OBJ), PROP=Prescaler")
     field( DESC, "EVG Mux Prescaler RB")
-    field( FLNK, "$(SYS){$(D)}ResetMxc-Cmd")
+    field( FLNK, "$(SYS):$(D):ResetMxc-Cmd.PROC")
```

Comment:
The preferred way to chase out NSLS2 naming convention out of .db files is with a macro to avoid breaking `vme-evg230-nsls2.substitutions` and other existing subtitutions.

Also, why the addition of `.PROC`?

Reply:
I didn't intend to change the pvname here. I added the `.PROC` field to quite a warning message.

Reply:
In this instance the "Forward-link uses Channel Access ..." warning is actually an error.  This link is intended to be local.

Reply:
OK.